pkg/api/constants.go

```diff
@@ -45,22 +45,22 @@ const (
 	AnnotationKeyPodSchedulingSpec = GroupName + "/pod-scheduling-spec"
 
 	// To leverage this scheduler, if one container in the Pod want to use the
-	// allocated GPUs for the whole Pod, it should contain below env.
+	// allocated leaf cells for the whole Pod, it should contain below env.
 	//   env:
 	//   - name: NVIDIA_VISIBLE_DEVICES
 	//     valueFrom:
 	//       fieldRef:
-	//         fieldPath: metadata.annotations['hivedscheduler.microsoft.com/pod-gpu-isolation']
+	//         fieldPath: metadata.annotations['hivedscheduler.microsoft.com/pod-leaf-cell-isolation']
 	// The annotation referred by the env will be populated by scheduler when bind the pod.
 	//
 	// Notes:
-	// 1. The scheduler directly delivers GPU isolation decision to
+	// 1. The scheduler directly delivers leaf cell isolation decision to
 	//    nvidia-container-runtime through Pod Env: NVIDIA_VISIBLE_DEVICES.
-	// 2. If multiple containers in the Pod contain the env, the allocated GPUs are
+	// 2. If multiple containers in the Pod contain the env, the allocated leaf cells are
 	//    all visible to them, so it is these containers' freedom to control how
-	//    to share these GPUs.
-	EnvNameNvidiaVisibleDevices  = "NVIDIA_VISIBLE_DEVICES"
-	AnnotationKeyPodGpuIsolation = GroupName + "/pod-gpu-isolation"
+	//    to share these leaf cells.
+	EnvNameNvidiaVisibleDevices       = "NVIDIA_VISIBLE_DEVICES"
```

Comment:
As the NVIDIA_VISIBLE_DEVICES is not used in hived code, can you move it to user manual doc?
Such as section:
Scheduling GPUs
1. NVidia GPUs
NVIDIA_VISIBLE_DEVICES and its comment here.

1. AMDGPUs
XXXX_VISIBLE_DEVICES and its comment here similar?

Reply:
As some places in the comment, still use gpu for NVIDIA_VISIBLE_DEVICES  is better.

---
In reply to: [460640199](https://github.com/microsoft/hivedscheduler/pull/28#discussion_r460640199) [](ancestors = 460640199)

Reply:
updated